.github/workflows/docker_release.yml

```diff
@@ -19,12 +19,20 @@ jobs:
       - name: Set env
         id: set_tag_version
         run: |
-          if [ "${{ github.event.client_payload.tagversion }}" != "" ]; then
-          echo ::set-output name=tagversion::${{ github.event.client_payload.tagversion }}
+          #Checks for a valid tag when the tag is manually pushed
+          if [[ $GITHUB_REF != "" && "${{ github.event.client_payload.tagversion }}" == "" ]]; then
+          echo ::set-output name=tagversion::$(egrep -o '(v[0-9]+\.[0-9]+\.[0-9]+)(-.*)?' <<< $GITHUB_REF)
           else
-          echo ::set-output name=tagversion::$(echo $GITHUB_REF | cut -d / -f 3)
+          #Runs when a tag is triggered automatically
+          echo ::set-output name=tagversion::${{ github.event.client_payload.tagversion }}
           fi
           echo $tagversion
+      - name: Check Tag Version
+        run: |
+          if [[ "${{ steps.set_tag_version.outputs.tagversion }}" == ""  ]]; then
+          echo "::error::Incorrect tag provided, please provide a tag with the correct format and try again"
+          exit 1
+          fi
       - name: Docker login
         run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login --username ${{ secrets.DOCKER_USER }} --password-stdin
       - name: Build production docker image
```

Comment:
we need to ensure that the `v` does get included in the docker image tag

Reply:
updated a57bd897af1b8e6e4c61480dccc6cab03f7db34c